### ISSUE TITLE
The reason for the test failure in Java 19+ is that the class to be mocked is now a sealed class.

### ISSUE THREAD

**Issue by hazendaz** [hazendaz/jmockit1#163]
The reason for the test failure in Java 21 is that the class to be mocked is now a sealed class.
The only solution seems to be to ignore the test.

- JDK18 or  earlier
```java
public class InetAddress implements java.io.Serializable {
```

- JDK19 or later
```java
public sealed class InetAddress implements Serializable permits Inet4Address, Inet6Address {
```

_Originally posted by @yukkes in https://github.com/hazendaz/jmockit1/issues/158#issuecomment-1837219874_

**Comment by hazendaz** [comment]
To do this using junit to selectively disable bad tests, we should move more of the tests to junit 5.  I'm working this issue separately and will for now allow jdk 21 to continue failing (really 19 or later).

This replaces #73

**Comment by hazendaz** [comment]
resolved by #210
